flair/embeddings/token.py

```diff
@@ -119,16 +120,21 @@ def get_named_embeddings_dict(self) -> Dict:
 class WordEmbeddings(TokenEmbeddings):
     """Standard static word embeddings, such as GloVe or FastText."""
 
-    def __init__(self, embeddings: str, field: str = None):
+    def __init__(self, embeddings: str, field: str = None, fine_tune: bool = False, force_cpu: bool = True,
+                 stable: bool = False):
         """
         Initializes classic word embeddings. Constructor downloads required files if not there.
         :param embeddings: one of: 'glove', 'extvec', 'crawl' or two-letter language code or custom
         If you want to use a custom embedding file, just pass the path to the embeddings as embeddings variable.
+        set stable=True to use the stable embeddings as described in https://arxiv.org/abs/2110.02861
         """
         self.embeddings = embeddings
 
         self.instance_parameters = self.get_instance_parameters(locals=locals())
 
+        if fine_tune and force_cpu and flair.device.type != "cpu":
+            raise Exception("Cannot train WordEmbeddings on cpu if the model is trained on gpu")
```

Comment:
And one more minor thing: Perhaps in the error message you could point to either setting `force_cpu=False` to enable training on GPU?

Reply:
sounds good